release-notes/major12/12.0.0/valtimo-frontend-libraries.md

```diff
@@ -27,9 +27,9 @@ The following features were added:
 
   More details about creating document definitions can be found [here](/using-valtimo/document/create-document-definition.md)
 
-* **Zaken-api plugin action end Hersteltermijn**
+* **Zaken-api plugin action start and end Hersteltermijn**
 
-  A Zaken-API plugin action has been added for ending a recovery period
+  A Zaken-API plugin action has been added for both starting and ending a recovery period.
```

Comment:
Can we have a link to the page where these actions are documented?

Reply:
added